analyses/snv-callers/README.md

```diff
@@ -5,7 +5,24 @@ This consensus mutation file is [MAF-like](#consensus-mutation-call) meaning it
 
 The resulting MAF and TMB files from this analysis are saved to `snv-callers/results/consensus/` folder and used by downstream analyses.
 
-See the comparison results plots [here](https://cansavvy.github.io/openpbta-notebook-concept/snv-callers/compare_snv_callers_plots.nb.html).
+Consensus mutations saved to `pbta-snv-consensus-mutation.maf.tsv.gz` are decided from agreement [3 out of 3 of the snv callers used](#02-merge_callersr).
+Four callers were originally run but VarDict was dropped from any downstream results because it suspiciously called very low VAF mutations that only it and none of the other three callers identified.
```

Comment:
```suggestion
Four callers were originally run but VarDict was dropped from downstream analysis because it called a large number of very low VAF mutations that were unique to it, suggesting that most of these were likely to be false positive calls.
```